pkg/api/resource/quantity.go

```diff
@@ -308,18 +324,16 @@ func (q *Quantity) String() string {
 //   +1 if q >  y
 //
 func (q *Quantity) Cmp(y Quantity) int {
-	num1 := q.Value()
-	num2 := y.Value()
-	if num1 < MaxMilliValue && num2 < MaxMilliValue {
-		num1 = q.MilliValue()
-		num2 = y.MilliValue()
+	if q.Amount == nil {
+		if y.Amount == nil {
```

Comment:
This is funny, I think this might have been a github CSS bug. Does it still look like it needs the indent to you? I saw the problem on my macbook, but now it looks correct to me (and is correct in my local copy).